components/contentselector/model/loadordererror.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef LOADORDERERROR_HPP
+#define LOADORDERERROR_HPP
+
+#include <QString>
+
+namespace ContentSelectorModel
+{
+    /// \Details of a suspected Load Order problem a plug-in will have. This is basically a POD
+    class LoadOrderError
+    {
+    public:
+        enum ErrorCode
+        {
+            ErrorCode_None               = 0,
+            ErrorCode_MissingDependency  = 1,
+            ErrorCode_InactiveDependency = 2,
+            ErrorCode_LoadOrder          = 3
+        };
+
+        inline LoadOrderError() : mErrorCode(ErrorCode_None) {};
+        inline LoadOrderError(ErrorCode errorCode, QString fileName)
+        {
+            mErrorCode = errorCode;
+            mFileName = fileName;
+        }
+        inline ErrorCode errorCode() const { return mErrorCode; }
+        inline QString fileName() const { return mFileName; }
+        bool operator==(const LoadOrderError& rhs) const;
+        QString toolTip() const;
+
+        /// \Sentinal to represent a "No Load Order Error" condition
```

Comment:
It was meant to be a doxygen command.  However I did not understand doxygen well enough.
I'll remove them for the time being.

And yes, my spelling is awful.  I'll fix.